css/main.css

```diff
@@ -191,24 +183,133 @@ h3 {
     font-family: Georgia, 'Times New Roman', Times, serif;
 }
 
-/* Sticky elements for Menu page */
+.navbar.sticky-top{
+    z-index: 1100;
+}
+.accordion-header.sticky-top {
+    top: 180px;
+} 
 
-/* ?? wondering if this media query is part of the answer for adjusting accordion header sticky height after hamburger menu is toggled ?? 
+/* beginning of CSS for specials */
+.column {
+    float: left;
+    width: 50%;
+}
 
-@media only screen and (max-width: 980px){
- .navbar-collapse .collapse .show {
- 
-    then adjust .accordion-header.sticky-top{ 
-        top: 250px;
+#Specials {
+    position: absolute;
+    left: auto;
+    text-align: left;
+    font-family:serif;
+}
 
-    }  ??
- }
+#socialMedia{
```

Comment:
#socialMedia css here is a duplicate of line 54 of this file. Also we won't need either once #55 is merged into master

Reply:
fixed